storm-server/src/main/java/org/apache/storm/localizer/AsyncLocalizer.java

```diff
@@ -261,6 +257,9 @@ private LocalizedResource getUserFile(String user, String key) {
     }
 
     private CompletableFuture<Void> downloadOrUpdate(Collection<? extends LocallyCachedBlob> blobs) {
+
+        final long remoteBlobstoreModTime = getRemoteBlobstoreUpdateTime();
```

Comment:
nit: change be changed to `remoteBlobstoreModTime` to `remoteBlobstoreUpdateTime`